apps/emqx_bridge_kafka/src/emqx_bridge_kafka_token_cache.erl

```diff
@@ -0,0 +1,230 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2025 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%--------------------------------------------------------------------
+-module(emqx_bridge_kafka_token_cache).
+
+-behaviour(gen_server).
+
+%% API
+-export([
+    start_link/0,
+
+    create_tables/0,
+    get_or_refresh/2,
+    unregister/1,
+    clear_cache/0,
+    clear_cache/1
+]).
+
+%% `gen_server' API
+-export([
+    init/1,
+    terminate/2,
+    handle_call/3,
+    handle_cast/2,
+    handle_info/2
+]).
+
+-include("emqx_bridge_kafka_internal.hrl").
+-include_lib("snabbkaffe/include/trace.hrl").
+
+%%------------------------------------------------------------------------------
+%% Type declarations
+%%------------------------------------------------------------------------------
+
+-define(registered, registered).
+-define(timers, timers).
+
+-define(KEY(CLIENT_ID), CLIENT_ID).
```

Comment:
is client ID unique ? can two connectors towards different kafka server share the same client ID (different uri etc.).